src/selectivity-multiple.js

```diff
@@ -329,6 +332,13 @@ var callSuper = Selectivity.inherits(MultipleSelectivity, {
     /**
      * @private
      */
+    _close: function() {
+        this.close();
+    },
```

Comment:
Seems this method is not really needed (the event could be hooked to `close()` directly).